Src/Private/Get-AbrOntapVserverS3Summary.ps1

```diff
@@ -0,0 +1,57 @@
+function Get-AbrOntapVserverS3Summary {
+    <#
+    .SYNOPSIS
+    Used by As Built Report to retrieve NetApp ONTAP Vserver S3 information from the Cluster Management Network
+    .DESCRIPTION
+
+    .NOTES
+        Version:        0.3.0
+        Author:         Jonathan Colon
+        Twitter:        @jcolonfzenpr
+        Github:         rebelinux
+    .EXAMPLE
+
+    .LINK
+
+    #>
+    [CmdletBinding()]
+    param (
+    )
+
+    begin {
+        Write-PscriboMessage "Collecting ONTAP Vserver S3 information."
+    }
+
+    process {
+        $VserverData = Get-AbrOntapApi -uri "/api/protocols/s3/services?"
+        $VserverObj = @()
+        if ($VserverData) {
+            foreach ($Item in $VserverData) {
+                $inObj = [ordered] @{
+                    'Vserver' = $Item.Name
+                    'HTTP' = $Item.is_http_enabled
+                    'HTTP Port' = $Item.port
+                    'HTTPS' = $Item.is_https_enabled
+                    'HTTPS Port' = $Item.secure_port
+                    'Status' = switch ($Item.enabled) {
+                        'True' { 'UP' }
+                        'False' { 'Down' }
```

Comment:
Fix casing

Reply:
I don't understand this comment

Reply:
Letter casing is inconsistent. `UP` is in uppercase and `Down` is in mixed case. Change `UP` to `Up`.